tests/integration/test_pagure.py

```diff
@@ -11,11 +11,13 @@
     os.path.dirname(os.path.realpath(__file__)), DATA_DIR
 )
 
+LAST_GENERATED_BY = "lachmanfrantisek"
```

Comment:
lol :D

Reply:
There is a problem that when we run the tests without user-token, we need to know the owner of the fork we want to check. (=user who pregenerated the files)

I would like to improve that -- I'll create an issue.

Reply:
https://github.com/packit-service/ogr/issues/181